hawkular-bus-common/src/main/java/org/hawkular/bus/common/MessageProcessor.java

```diff
@@ -338,42 +339,6 @@ public MessageId sendWithBinaryData(ProducerConnectionContext context, BasicMess
     }
 
     /**
-     * Same as {@link #sendRPC(ProducerConnectionContext, BasicMessage, Class, Map)} with <code>null</code> headers.
-     */
-    public <R extends BasicMessage> ListenableFuture<BasicMessageWithExtraData<R>> sendRPC(
```

Comment:
so are we getting rid of the RPC-like (synchronous) messaging entirely from the hawkular bus API?